src/test/cypress/integration/ProgrammingExerciseStaticCodeAnalysis.spec.ts

```diff
@@ -0,0 +1,76 @@
+import scaSubmission from '../fixtures/programming_exercise_submissions/static_code_analysis/submission.json';
+import { artemis } from '../support/ArtemisTesting';
+import { makeSubmissionAndVerifyResults, startParticipationInProgrammingExercise } from '../support/pageobjects/OnlineEditorPage';
+
+// The user management object
+const users = artemis.users;
+
+// Requests
+const courseManagement = artemis.requests.courseManagement;
+
+// PageObjects
+const editorPage = artemis.pageobjects.programmingExercise.editor;
+const scaConfig = artemis.pageobjects.programmingExercise.scaConfiguration;
+const scaFeedback = artemis.pageobjects.programmingExercise.scaFeedback;
+
+describe('Static code analysis tests', () => {
+    let course: any;
+    let exercise: any;
+
+    before(() => {
+        setupCourseAndProgrammingExercise();
+    });
+
+    it('Configure every static code analysis category to influence grading', () => {
+        cy.login(users.getAdmin());
+        scaConfig.visit(course.id, exercise.id);
+        scaConfig.makeEveryScaCategoryInfluenceGrading();
+        scaConfig.saveChanges();
+    });
+
+    it('Makes successful submission with SCA errors', function () {
+        startParticipationInProgrammingExercise(course.title, exercise.title, users.getStudentOne());
+        makeSuccessfulSubmissionWithScaErrors();
+    });
+
+    after(() => {
+        if (!!course) {
+            cy.login(users.getAdmin());
+            courseManagement.deleteCourse(course.id);
+        }
+    });
+
+    /**
+     * Creates a course and a programming exercise inside that course.
+     */
+    function setupCourseAndProgrammingExercise() {
+        cy.login(users.getAdmin());
+        courseManagement.createCourse().then((response) => {
+            course = response.body;
+            courseManagement.addStudentToCourse(course.id, users.getStudentOne().username);
+            courseManagement
+                .createProgrammingExercise({ course }, 50)
+                .its('body')
```

Comment:
Since you replaced it in another file I'd suggest moving the body down in the then

Reply:
True, I think I am not really consistent with that. I will move it into the then block